packages/ui/src/components/Hero/Hero.tsx

```diff
@@ -105,3 +96,20 @@ export function Hero({
         </div>
     );
 }
+
+export namespace Hero {
+    export type DisplayedCategory = CategoriesList.DisplayedCategory;
+
+    export interface Props {
+        story: StoryWithImage;
```

Comment:
1. From what I see, this does not need the whole Story object. Only `title`, `subtitle` and `slug` are used.
2. URL generation should not be a concern of this component.

That being said, I suggest declarign a local type here: `DisplayedStory` and use it in `Props`:

```ts
export interface DisplayedStory {
   title: Story['title'];
   subtitle: Story['subtitle'];
   href: string;
}
```